Makefile

```diff
@@ -101,20 +89,20 @@ $(LIBBPF_INCLUDES): $(LIBBPF_SYS_LIBBPF_INCLUDES)
 	cp $^ $(LIBBPF_INCLUDES)/bpf/
 endif
 
-$(OVS_HOOKS): INCLUDES_EXTRA := -I$(OVS_INCLUDES)
+$(OVS_HOOK): INCLUDES_EXTRA := -I$(OVS_INCLUDES)
```

Comment:
Instead of special-casing OvS, could we automatically add `-Isrc/module/<name>/bpf/include` (if path exists) when building each module?